modules/core/04-channel/keeper/upgrade_test.go

```diff
@@ -1305,10 +1305,10 @@ func (suite *KeeperTestSuite) TestAbortHandshake() {
 
 			tc.malleate()
 
-			err := channelKeeper.AbortUpgrade(suite.chainA.GetContext(), path.EndpointA.ChannelConfig.PortID, path.EndpointA.ChannelID, upgradeError)
```

Comment:
guess we could also define the function as a var and trim down on the `NotPanics/Panics` calls in a similar vein. Leaving to your discretion.